Lesson Plans/Flight Instructor/CFI/CFI1.4 Teaching Process.md

```diff
@@ -0,0 +1,250 @@
+2. Key Theory
+	1. Covers the key points & concepts
+	2. Straightforward and factual
+	3. Best if relatively simple
+3. Reasons for the use
+	1. Help gain and hold learners' attention
+	2. Help with understanding and retention
+	3. Solves certain language barrier problems
+	4. Clarify relationships between material objects and concepts
+	5. Saves time
+4. [[Guideline for Use of Instructional Aids]]
+5. Types of Instructional Aids
+	1. Marker Board
+	2. Supplemental Printed Material
+	3. Enhanced Training Materials
+	4. Projected Material
+	5. Video
+	6. Interactive Systems
+	7. Computer-Assisted Learning
+	8. Models, Mockups and Cut-Aways
+
+#### Integrated Flight Instruction
+1. Flight maneuvers is taught to performed by both visual & instrument references
+	1. Develops habits of continuously monitoring their own and aircraft's performance
+	2. Gains better operating efficiency and aircraft performance through more accurate flying
+2. Procedures
+	1. Start from the beginning: Includes both expected instrument indications to briefing and outside references
+	2. Constantly integrated into all lessons
+3. See & Avoid
+	1. Make sure learners develop the habit of scanning for traffic at all times
+	2. Do not let them assume instructors are assuming all responsibilities for traffic avoidance
+
+#### Planning Instructional Activity
+1. Blocks of Learning
+	1. Constitute necessary parts of the total objective; see [[~ PPL Blocks of Learning.png]] for an example
+	2. They denote units of learning which can be measured and evaluated and then build upon, not sequentially.
+		1. > [!example] Blocks for Pre-solo
+		   > An example of pre-solo building blocks can look like:
+		   > ![[Pre-solo Blocks of Learning.png]]	   
+    3. Blocks should be truly integral parts, extraneous blocks are expensive frills and detract from the final objective
+    4. Give learners a sense of accomplishment and boost in self-confidence everytime they complete a block. Each one of the block is also more manageable.
+2. Training Syllabus
+	1. Practical guide for logical sequencing of the lesson and ensuring all requirements and completed and properly documented
+	2. A brief yet comprehensive outline of most efficient way of training
+	3. Relies on the blocks of learning, establishing objectives and standards for the overall course, separate ground and flight segments, recommended class time and each stage of training
+	4. Using a Training Syllabus
+		1. It should be flexible and used primarily as a guide
+			1. Order of training should be modified to adapt learner's prior experience or other demands
+			2. Instructor must consider the effects on learning blocks when departing from the syllabus
+		2. Ground Lessons
+			1. Focus on the cognitive domain of learning
+			2. Instructor should stress [[ADM]], safety factors with traditional aviation subjects to favorably influence a learner's attitude, beliefs and values
+		3. Flight Lessons
+			1. Emphasize on the psychomotor domain
```

Comment:
[[Domains of Learning#Psychomotor Domain|psychomotor domain]]

Reply:
love these links